tox.ini

```diff
@@ -104,21 +104,44 @@ commands =
 description = run mypy (static type checker)
 
 deps =
-    mypy>=0.770
+    mypy>=0.902
     mypy-zope
     typing-extensions
     ; Libraries which include type annotations:
     hypothesis
     pytest>=6.0.0
     sphinx>=3.4.0
     git+https://github.com/twisted/twisted.git
+    types-requests
+    types-docutils
 
 commands =
     mypy                                       \
         --cache-dir="{toxworkdir}/mypy_cache"  \
         {tty:--pretty:}                        \
         {posargs:pydoctor docs/epytext_demo}
 
+[testenv:mypy-docutils-stubs]
+description = run mypy with docutils-stubs (does not pass for now)
+; See: https://github.com/python/typeshed/issues/1269
```

Comment:
Do we want an issue for this repo as well, to track turning it on in CI when this is done?

Reply:
See https://github.com/twisted/pydoctor/issues/409